unlock-app/src/hooks/useCheckoutCommunication.ts

```diff
@@ -197,6 +196,10 @@ export const useCheckoutCommunication = () => {
 
   if (useDelegatedProvider && !providerAdapter) {
     setProviderAdapter({
+      parentOrigin: () => {
+        // @ts-expect-error Property 'parentOrigin' does not exist on type 'ChildAPI'.ts(2339)
```

Comment:
🤔

Reply:
yeah postmates does not have good TS support!